configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg

```diff
@@ -20,5 +20,7 @@ if exec xrootd
    xrd.port 1094
 fi
 
+# This is the configuration for a cache
+all.role server
```

Comment:
Does this work for the two cases that we want to support?

- A single XCache host
- Multiple caches behind a local redirector

Reply:
Yes role should always be set up. It shoudl not only be set up for redirectors. I have it in all my [standaldone stashcaches](https://github.com/opensciencegrid/prp-stashcache/blob/master/stashcache-server.cfg#L35) so I know it works

Reply:
In that case, could you delete this line? https://github.com/opensciencegrid/xcache/blob/master/configs/cms-xcache/config.d/90-cms-xcache-local-redirector.cfg#L16